### PR TITLE
[API Management] Add parameter option "GraphQL" to ImportApi cmdlet 

### PR DESCRIPTION
**Pull request by wpooley**
## Description

Add parameter option "GraphQL" to ImportApi cmdlet 

## Checklist

- [X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/ApiManagement/ApiManagement/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Support GraphQL Specification Format
```
Comment (BethanyZhou) [inline src/ApiManagement/ApiManagement/ChangeLog.md:21]:
```suggestion
* Supported GraphQL Specification Format
```

File: src/ApiManagement/ApiManagement/help/Import-AzApiManagementApi.md [hunk 2]
```diff
@@ -142,6 +142,14 @@ ServiceName                   : contoso
 
 This command imports an API from the specified Open 3.0 specification document and create a new ApiVersion.
 
+### Example 7: Import an API from a GraphQL link
+```powershell
+$ApiMgmtContext = New-AzApiManagementContext -ResourceGroupName "Api-Default-WestUS" -ServiceName "contoso"
+Import-AzApiManagementApi -Context $ApiMgmtContext -SpecificationFormat "GraphQL" -SpecificationUrl "http://contoso.com/graphql" -Path "apis"
+```
+
```
Comment (BethanyZhou) [inline src/ApiManagement/ApiManagement/help/Import-AzApiManagementApi.md:150]: please add sample output if possible with following pattern:
\```output
{output}
\```

**Comment by isra-fel** [comment]
Kindly ping @wpooley

File: src/ApiManagement/ApiManagement/help/Import-AzApiManagementApi.md (hunk 2 above)
Comment (wpooley) [reply src/ApiManagement/ApiManagement/help/Import-AzApiManagementApi.md:177]: Added sample output

**Comment by wpooley** [comment]
Sorry for the delay, made the changes @isra-fel  / @BethanyZhou

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**